app/api/crud.py

```diff
@@ -82,3 +82,29 @@ async def get(
         cross_node_results += response.json()
 
     return cross_node_results
+
+async def get_terms_labels(data_element_URI: str, labels: bool):
+    """Makes GET requests to one or more Neurobagel node APIs using httpx where the parameter is a Neurobagel variable."""
+
+    cross_node_results = []
+
+    for node_url in util.parse_nodes_as_list(util.NEUROBAGEL_NODES):
+        if labels:
+            url = node_url + "attributes/"+data_element_URI+"/vocab"
+        else:
+            url = node_url + "attributes/"+data_element_URI
+        response = httpx.get(
+            url=url,
+            # Enable redirect following (off by default) so APIs behind a proxy can be reached
+            follow_redirects=True,
+        )
+
+        if not response.is_success:
+            raise HTTPException(
+                status_code=response.status_code,
+                detail=f"{response.reason_phrase}: {response.text}",
+            )
+        
+        cross_node_results.append(response.json())
```

Comment:
This part here is pretty duplicate-y compared with the `get` method above. How about we factor that part out into its a generic `get` method and have the URL making business be taken care of by more aptly named functions?